go.mod

```diff
@@ -12,6 +12,7 @@ go 1.23.6
 require (
 	github.com/DataDog/zstd v1.5.2
 	github.com/NYTimes/gziphandler v1.1.1
+	github.com/StephenButtolph/canoto v0.10.0
```

Comment:
I'm a bit torn on if we should wait for `v1.0.0` before integrating (I think adding maps and https://github.com/StephenButtolph/canoto/issues/75 are all we'd want to do before calling it `v1`)

Reply:
I don't have a strong opinion, but this is blocking updating HyperSDK to the latest commit of AvalancheGo, which is required to use the latest tmpnet work.

Reply:
no need to wait